src/Queue/CreateJob.js

```diff
@@ -257,10 +234,11 @@ function handleSubmit(){
     status : 'submitted',
     code : mymodel,
     command : command,
-    hardware_config : config,
+    hardware_config : hardwareConfig,
```

Comment:
I have a problem using the deployed version and also with this branch. I can't submit a job. By playing around a bit, the issue seems to come from `hardwareConfig`, however I don't see any difference with the previous code...

Reply:
more precisely, if I comment this line, it works :thinking:

Reply:
should be fixed now

Reply:
One thing i don't understand: the joblist contains your last jobs from today. So, the code should work. However on my side I have the following error:
`SyntaxError: Unexpected end of JSON input`

Reply:
that probably means there's a JSON syntax error in the hardwareConfig box. In the next release we should add an error message for the user. What did you put in the hardwareConfig box?

Reply:
I let the field empty, as was the case in most tests before, or exactly as the suggested example

Reply:
ok, good point, I'll fix that now. This is why we need unit tests!

Reply:
I am all the more perplexed that looking at the code, the modifications made here don't change the structure in any way, since only objects related to the hardware config have been renamed 🤔